### PR TITLE
Jave Extension Guide - Frontend

### PR DESCRIPTION
**Pull request by akaroml**
Signed-off-by: Rome Li <rome.li@microsoft.com>

Attach screenshot to make more sense.
image

### CODE REVIEW

File: src/ext-guide/assets/index.html [hunk 1]
```diff
@@ -0,0 +1,253 @@
+<!DOCTYPE html>
```
Comment (Eskibear) [inline src/ext-guide/assets/index.html:1]: strongly request GitHub to support HTML preview, just like markdown...

File: src/ext-guide/index.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  webviewPanel.iconPath = vscode.Uri.file(path.join(context.extensionPath, "logo.lowres.png"));
+  const resourceUri = context.asAbsolutePath("./out/assets/ext-guide/index.html");
+  webviewPanel.webview.html = await loadTextFromFile(resourceUri);
+
+  context.subscriptions.push(webviewPanel.onDidDispose(onDisposeCallback));
+  context.subscriptions.push(webviewPanel.webview.onDidReceiveMessage(async (e) => {
+    if (e.command === "tabActivated") {
+      let tabId = e.tabId;
+      sendInfo(operationId, {
+        infoType: "tabActivated",
```
Comment (Eskibear) [inline src/ext-guide/index.ts:42]: don't forget GDPR tagging

File: src/ext-guide/assets/index.html (hunk 1 above)
Comment (akaroml) [reply src/ext-guide/assets/index.html:1]: git pull and F5 😄

File: src/ext-guide/index.ts (hunk 2 above)
Comment (Eskibear) [reply src/ext-guide/index.ts:42]: done.

**Pull request merged**